docs/architecture/cloud-native/candidate-apps.md

```diff
@@ -1,52 +1,57 @@
 ---
 title: Candidate Apps for Cloud Native
-description: Architecting Cloud Native .NET Apps for Azure | Candidate Apps for Cloud Native
-ms.date: 06/30/2019
+description: Architecting Cloud-Native .NET Apps for Azure | Candidate Apps for Cloud Native
+author: robvet
+ms.date: 07/28/2019
 ---
 # Candidate Apps for Cloud Native
```

Comment:
```suggestion
# Candidate apps for cloud native
```

Reply:
Thinking... this would be the correct formatting?

---
title: Candidate apps for cloud native
description: Architecting Cloud-Native .NET Apps for Azure | Candidate apps for cloud native
author: robvet
ms.date: 08/14/2019
---
# Candidate apps for cloud native

Reply:
That looks about right.

Just description would be something like
"description: Learn which types of applications benefit the most from a cloud-native architecture and what to consider when moving to this approach."